index.html

```diff
@@ -0,0 +1,204 @@
+                </ul>
+                <button class="see-project">See Project</button>
+            </div>
+        </div>
+        <!-- End of Project 3 -->
+
+        <!-- Start of Project 4 -->
+        <div id="tonic-project">
+            <div class="data">
+                <img src="./images/snapshots/Snapshoot Portfolio 4.png" alt="Tonic project snapshot">
+                <h2 class="project-title">Multi-Post <br />Stories</h2>
+                <div class="work-nature">
+                    <label class="canopy">CANOPY</label>
+                    <div class="bullet-point"><img src="./images/icons/Counter.png" alt=""></div>
+                    <label class="backend"> Back End Dev</label>
+                    <div class="bullet-point"><img src="./images/icons/Counter.png" alt=""></div>
+                    <label class="year">2015</label>
+                </div>
+                <p class="work-description">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                <ul class="tools">
+                    <li><label class="tool-press">HTML</label></li>
+                    <li><label class="tool-press">CSS</label></li>
+                    <li><label class="tool-press">JAVASCRIPT</label></li>
+                </ul>
+                <button class="see-project">See Project</button>
+            </div>
+        </div>
+        <!-- End of Project 4 -->
+    </div>
+    <section id="skills-page">
+        <div class="about-description">
+            <h2 class="main-text">About Myself</h2>
+            <p class="main-para">Hello I’m a software developer! I can help you build a product , feature or website. Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.</p>
+            <label for="" class="connect-message">LET'S CONNECT</label>
+        
+            <div id="link-media">
+                <ul class="social-icons">
+                    <li>
+                      <a href="#" class="logos"><img src="./images/icons/twitter.svg" alt="Twitter" /></a>
+                    </li>
+                    <li>
+                      <a href="#" class="logos"><img src="./images/icons/Linkedin.png" alt="LinkedIn" /></a>
+                    </li>
+                    <li>
+                      <a href="#" class="logos"><img src="./images/icons/m.svg" alt="Medium" /></a>
+                    </li>
+                    <li>
+                      <a href="#" class="logos"><img src="./images/icons/github.svg" alt="GitHub" /></a>
+                    </li>
+                    <li>
+                      <a href="#" class="logos"><img src="./images/icons/random.svg" alt="Random" /></a>
+                    </li>
+                </ul>
+            </div>
+            <button class="resume-link">Get my resume</button>
+        </div>
+        <div class="skills-block">
```

Comment:
- [ ] As suggested by the previous reviewer, your list of technology section is still missing the arrows on the right. Kindly add those arrows so that your design matches the figma designs. Thos arrows are basically there to toggle the technology list 👍🏽 

your design 👇🏽 
![image](https://user-images.githubusercontent.com/63473584/227702280-a30c87dc-33c5-4b10-b34b-956a784a3d06.png)

figma design 👇🏽 
![image](https://user-images.githubusercontent.com/63473584/227702315-0b121d67-b9ab-49a5-b6e5-9e120d621cf6.png)